invenio_theme/assets/semantic-ui/js/invenio_theme/theme.js

```diff
@@ -14,7 +14,42 @@ import "semantic-ui-less/semantic.less";
 
 // Initialize Semantic UI components
 jquery(".ui.dropdown").dropdown();
-jquery(".ui.accordion").accordion();
+
+jquery(".ui.accordion").accordion({
+  selector: {
+    trigger: '.trigger'
+  },
+  onOpening: function() {
+    const $accordionTrigger = this.closest('.ui.accordion').find('.trigger');
```

Comment:
should work with, to be tested
```suggestion
    const $accordionTrigger = $(this).find('.trigger');
```

Reply:
No, as `this` is actually the `.content`.